frameworks/cassandra/tests/test_backup_and_restore.py

```diff
@@ -106,3 +113,98 @@ def test_backup_and_restore_to_s3() -> None:
         plan_parameters,
         config.get_foldered_node_address(),
     )
+
+
+@pytest.mark.aws
+@pytest.mark.sanity
+def test_backup_and_restore_to_s3_compatible_storage() -> None:
+    try:
+        sdk_install.install(
+            "minio",
+            "minio",
+            expected_running_tasks=0,
+            package_version="0.0.13-RELEASE.2018-10-06T00-15-16Z",
+            wait_for_deployment=False,
+        )
+        temp_key_id = os.getenv("AWS_ACCESS_KEY_ID")
+
+        if not temp_key_id:
+            assert (
+                False
+            ), 'AWS credentials are required for this test. Disable test with e.g. TEST_TYPES="sanity and not aws"'
+        temp_secret_Access_key = os.getenv("AWS_SECRET_ACCESS_KEY")
+        is_strict = sdk_utils.is_strict_mode()
+        if is_strict:
+            sdk_security.create_service_account(
+                service_account_name="marathon-lb-sa",
+                service_account_secret="marathon-lb/service-account-secret",
+            )
+            sdk_cmd.run_cli(
+                "security org users grant marathon-lb-sa dcos:service:marathon:marathon:services:/ read"
+            )
+            sdk_cmd.run_cli(
+                'security org users grant marathon-lb-sa dcos:service:marathon:marathon:admin:events read --description "Allows access to Marathon events"'
+            )
+            options = {
+                "marathon-lb": {
+                    "secret_name": "marathon-lb/service-account-secret",
+                    "marathon-uri": "https://marathon.mesos:8443",
+                }
+            }
+
+            sdk_install.install(
+                "marathon-lb",
+                "marathon-lb",
+                expected_running_tasks=0,
+                additional_options=options,
+                package_version="1.14.0",
+                wait_for_deployment=False,
```

Comment:
since `options` are really the only thing changing, you could instead create an empty dict before `is_strict` and use [sdk_utils.merge_dictionaries](https://github.com/mesosphere/dcos-commons/blob/master/testing/sdk_utils.py#L234) to combine the options you're adding above. Then you can remove the else on L164 call `sdk_install(additional_options=options)` no matter what.